voltdbclient/url_test.go

```diff
@@ -23,28 +23,33 @@ func TestParseURL(t *testing.T) {
 	}
 
 	for _, s := range sample {
-		u, err := parseURL(s.conn)
-		if s.pass {
-			if err != nil {
-				t.Fatal(err)
+		s := s //capture variable
+		t.Run("", func(t *testing.T) {
```

Comment:
@mehmetalisavas I can't justify having `t.Run` in the test case, please see the comment above for details.